.gitignore

```diff
@@ -1 +1,5 @@
 git/
+ecs-params.yml
```

Comment:
ecs-params.yml contains cluster specific information for it to work properly, so if we want a solution that can manage multiple clusters, we need a way to dynamically build a proper ecs-params.yml for each cluster dynamically.  There's a command called get-params.sh that reads the right configuration info from ECS and builds a file called `cluster-name-ecs-params.yml`.  Subsequent commands then run using `cluster-name-ecs-params.yml` instead of `ecs-params.yml`.  That let's anyone bootstrap any cluster in their environment without having stale configuration info checked into source control.